### DISCUSSION TITLE
Setting for 7tv to not replace the normal emote button?

### DISCUSSION THREAD

**Discussion by blazex224** [SevenTV/Extension#381] [Q&A] [answered]
Is there a setting for 7tv to not replace the normal emote button? Using 3.0.0. It replaces the standard Twitch emote button entirely, which I don't like at all because having both is much more convenient. Wondering if there is a toggle for this, I cannot find one

**Comment by AnatoleAM** [comment, marked as answer]
This is already implemented. Use the [nightly](https://chrome.google.com/webstore/detail/7tv-nightly/fphegifdehlodcepfkgofelcenelpedj?referrer=7tv.app) branch if you want this functionality right now, or wait for the stable release of 3.0.1.
